rosdep/base.yaml

```diff
@@ -3330,6 +3329,7 @@ libgmock-dev:
   rhel: [gmock-devel]
   ubuntu:
     focal: [libgmock-dev]
+    jammy: [libgmock-dev]
```

Comment:
This rule looks to be valid for focal and newer. Please update this to use `'*': [libgmock-dev]` and `bionic: null`

Reply:
@tfoote updated at https://github.com/ros/rosdistro/pull/32306/commits/fa06ebc50bdfa64c19db509f033aa29ef212dd39